baseapp/abci.go

```diff
@@ -637,14 +638,15 @@ func (app *BaseApp) createQueryContext(height int64, prove bool) (sdk.Context, e
 			)
 	}
 
-	cacheMS, err := app.cms.CacheMultiStoreWithVersion(height)
+	version, err := app.store.GetVersion(height)
```

Comment:
let's add `CacheMultiStoreWithVersion` -  this way we will be compatible with `v1/Multistore` interface and we don't even need to rewrite it in v2

Reply:
replaced it with `version.CacheWrap`